CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc

```diff
@@ -178,9 +178,34 @@ HcalGainWidth HcalDbHardcode::makeGainWidth(HcalGenericDetId fId) const {  // Ge
   return result;
 }
 
-HcalPFCut HcalDbHardcode::makePFCut(HcalGenericDetId fId) const {  // GeV
+HcalPFCut HcalDbHardcode::makePFCut(HcalGenericDetId fId, double intLumi, bool noHE) const {  // GeV
+
+  // from SLHCUpgradeSimulations/Configuration/python/aging.py
+  double lumis[] = {300, 1000, 3000, 4500};  // integrated lumi points
```

Comment:
All these can be made `const` and moved inside the scope of the `if` block below

Reply:
@perrotta   you're right (I've sketched it on Saturday morning not much thinking...)